src/features/dashboard/components/Dialogs/DeleteAppDialog/index.tsx

```diff
@@ -10,52 +11,49 @@ type TDeleteAppDialogProps = {
 
 const DeleteAppDialog = ({ appId, onClose }: TDeleteAppDialogProps) => {
   const { deleteApp } = useDeleteApp();
+  const { deviceType } = useDeviceType();
 
   const onOpenChange = useCallback(
-    (open: boolean) => {
+    (open) => {
       if (!open) {
         onClose();
       }
     },
     [onClose],
   );
 
-  const actionButtons: TModalActionButton[] = useMemo(
-    () => [
-      {
-        id: 0,
-        text: 'Yes, delete',
-        color: 'primary',
-        onClick: () => {
-          deleteApp(appId);
-          onClose();
-        },
-      },
-      {
-        id: 1,
-        text: 'Cancel',
-        color: 'secondary',
-        onClick: () => {
-          onClose();
-        },
-      },
-    ],
-    [appId, deleteApp, onClose],
-  );
+  useEffect(() => {
+    document.body.style.overflow = 'hidden';
+    return () => {
+      document.body.style.overflow = '';
+    };
+  }, []);
 
   return (
-    <Modal defaultOpen onOpenChange={onOpenChange}>
-      <Modal.Portal>
-        <div className='modal-overlay'>
-          <Modal.Overlay />
-          <Modal.DialogContent
-            title='Delete app'
-            content='Are you sure you want to delete this app?'
-            action_buttons={actionButtons}
-            has_close_button
-          />
+    <Modal
+      isOpened={true}
+      toggleModal={onOpenChange}
+      primaryButtonLabel='Yes, delete'
+      secondaryButtonLabel='Cancel'
+      disableCloseOnOverlay
+      isMobile={deviceType !== 'desktop'}
+      showHandleBar
+      primaryButtonCallback={() => {
+        deleteApp(appId);
+        onClose();
+      }}
+      secondaryButtonCallback={onClose}
+      showSecondaryButton
+    >
+      <div className='icon-wrapper'>
+        <div className='modal-icon'>
```

Comment:
```suggestion
      <div className='icon-wrapper'>
        <div className='icon-wrapper__modal-icon'>
```

Use BEM convention, instead of writing 2 classes nest the 2nd class under the first class... this avoid conflicts between same styles.